docs/data/components/progress/UnstyledProgressIntroduction/css-modules/index.tsx

```diff
@@ -1,80 +1,21 @@
 'use client';
 import * as React from 'react';
-import { useTheme } from '@mui/system';
 import { Progress } from '@base-ui-components/react/progress';
+import classes from '../../styles.module.css';
 
 export default function UnstyledProgressIntroduction() {
   return (
-    <div className="App">
-      <Progress.Root className="Progress" value={50} aria-labelledby="ProgressLabel">
-        <span className="Label" id="ProgressLabel">
-          Uploading files
-        </span>
-        <Progress.Track className="Progress-track">
-          <Progress.Indicator className="Progress-indicator" />
-        </Progress.Track>
-      </Progress.Root>
-      <Styles />
-    </div>
+    <Progress.Root
+      className={classes.progress}
+      value={50}
+      aria-labelledby="ProgressLabel"
+    >
+      <span className={classes.label} id="ProgressLabel">
+        Uploading files
+      </span>
+      <Progress.Track className={classes.track}>
+        <Progress.Indicator className={classes.indicator} />
```

Comment:
This demo looks broken on CodeSandbox (the progress bar is invisible). Could you please check?

Reply:
![Screenshot 2024-11-29 at 1 11 24 PM](https://github.com/user-attachments/assets/6ff218ed-4f89-4ed8-bc8c-6c1f08570c4f)
Looks like the different CSS color vars I was using from different `theme.css` got lost because `theme.css` isn't in the sandbox 

I'll put fallback values for the stylesheet touched in this PR

Reply:
https://github.com/mui/base-ui/issues/894 FYI @michaldudak